### PR TITLE
BUG: option to avoid latex in qtconsole #12182

### LINKED ISSUE #12182: BUG: _repr_latex is called in jupyter qtconsole
xref #11778

image

### PR DESCRIPTION
**Pull request by chris-b1**
closes #12182

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
Thanks!

File: doc/source/whatsnew/v0.18.0.txt [hunk 1]
```diff
@@ -312,6 +312,24 @@ New Behavior:
...
+for conversion to latex in a ipython/jupyter notebook using nbconvert. (:issue:`11778`)
+
+Note that this must be activated by setting the option ``display.latex.repr`` to ``True`` (issue:`12182`)
+
+.. ipython:: python
+
+   df = pd.DataFrame({'a': [1, 2, 3]})
+   df._repr_latex_()
+   with pd.option_context('display.latex.repr', True):
+       print(df._repr_latex_())
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.18.0.txt]: I am wondering a bit if this example is useful. As you would never use this kind of code like that, so maybe it is a possibly confusing

Comment (chris-b1) [inline doc/source/whatsnew/v0.18.0.txt]: Yeah, I see what you're saying, it is contrived.  I was just trying to think of some example - for some reason I always forget the options syntax.  Maybe instead

```
For example, if you have a jupyter notebook you plan to convert to latex using nbconvert, place the statement ``pd.set_option('display.latex.repr', True)`` in the first cell to have the contained DataFrame output also stored as latex.
```

Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.18.0.txt]: Yes, I think that is more useful

**Comment by jreback** [comment]
thanks @chris-b1

**Pull request closed without merging**